gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -318,14 +318,14 @@ boolean cancelIfStale() {
         switch (this.state) {
           case IDLE:
             if (!idleTimeout.isZero() && waitTime >= idleTimeout.toMillis()) {
-              myError = new WatchdogTimeoutException("Canceled due to idle connection", false);
+              myError = new WatchdogTimeoutException("Canceled due to idle connection. idle timeout is " + idleTimeout.toMillis() + " ms", false);
```

Comment:
idle timeout is always 5 minutes, so I don't think we need to log it here.

Reply:
Sure, but the people seeing this log ling likely wouldn't know. And it's possible that we'll change it in the future?

Reply:
Fair. I don't think we ever want to let customer configure idleTimeout though.